conf/application.conf

```diff
@@ -58,7 +57,7 @@ microservice {
       host = graphite
       port = 2003
       prefix = play.${appName}.
-      enabled = true # disabled for local dev, should be enabled otherwise
+      enabled = false
```

Comment:
make enabled = true and add the comment line # disabled for local dev, should be enabled otherwise